source/agora/consensus/data/Transaction.d

```diff
@@ -99,6 +123,24 @@ public struct Transaction
             serializePart(output, dg);
     }
 
+    /// Transaction type serialize & deserialize for unittest
```

Comment:
Same comment, please move the unittests outside the class.

Reply:
Move to outside completed